dsl/git_push_test.jmx

```diff
@@ -0,0 +1,254 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jmeterTestPlan version="1.2" properties="2.8" jmeter="2.13 r1665067">
```

Comment:
The README.md documents how to add those variables in the Jmeter GUI - it's only for extra reference really, how about I just extract those relevant parts of the xml and put in the README.md for reference? It's just in case someone has some weird issues with the variables being miss set by the gui.